src/types.ts

```diff
@@ -75,6 +75,10 @@ export interface ServerResponse {
   sql_state: string;
 }
 
+export interface ServerRequest {
+  id: string;
```

Comment:
It would make sense to include `type` here at minimum as well since that is required in the request.

```suggestion
  id: string;
  type: string;
```